control-plane/rest/service/src/v0/snapshots.rs

```diff
@@ -90,7 +90,15 @@ impl apis::actix_server::Snapshots for RestApi {
             entries: snaps
                 .entries()
                 .iter()
-                .map(to_models_volume_snapshot)
+                .filter_map(|s| {
+                    if models::SpecStatus::from(s.meta().status().clone())
```

Comment:
This doesn't seem right, I think this should be done in the CSI side and not here

Reply:
@tiagolobocastro I've fixed to make changes to the csi now.